tests/common-tests.h

```diff
@@ -6364,6 +6370,381 @@ TEST_CASE_METHOD(common_tests, "Into XML vector with several fetches", "[core][x
     REQUIRE(!st.fetch());
 }
 
+TEST_CASE_METHOD(common_tests, "BLOB", "[core][blob]")
+{
+    soci::session sql(backEndFactory_, connectString_);
+
+    auto_table_creator tableCreator(tc_.table_creator_blob(sql));
+
+    if (!tableCreator.get())
+    {
+        try
+        {
+            soci::blob blob(sql);
+            FAIL("BLOB creation should throw, if backend doesn't support BLOBs");
+        } catch (const soci_error &)
```

Comment:
Minor style tweak, but in SOCI conventions this should be
```suggestion
        }
        catch (const soci_error &)
```